packages/vscode-extension/package.json

```diff
@@ -139,12 +139,12 @@
 				"command": "suitecloud.updateobject",
 				"key": "ctrl+alt+u",
 				"mac": "cmd+alt+u",
-				"when": "resource =~ //Objects/.+\\.xml$/"
+				"when": "resource =~ /\\/Objects\\/.+\\.xml$/"
```

Comment:
Are the changes here intentional?

Reply:
Those changes where in my workspace, I forgot to remove them.
They are the solution to not show the specific command in the context menu in any place as those `when` clauses are kind of broken.
I can leave them or remove them, as you wish. I did test them in my Windows machine and worked fine, but maybe is better to leave as it was.

Reply:
I think we already have a separate issue to work on that part. Maybe better to move these changes to that issue?

Reply:
I'll remove those changes then. 👍